schemas/Scenarios.yml

```diff
@@ -91,7 +91,11 @@ definitions:
         type: integer
         multipleOf: 1
         minimum: 0
-
+      
+      srcHospitalBeds:
+        type: string
+        minLength: 1
+      
```

Comment:
You are currently not setting srcHospitalBeds as required in the schema. This means the app will have to tolerate missing keys. I'm not sure if we should set it as required or not, we will need to see later.

Reply:
Yes, I wasn't sure if it was necessary to make it required or not given its tenuous inclusion at the moment, but it's only a 1 line change in code, so we can always account for it later. From my understanding (albeit rudimentary), not making it required wouldn't flag some sort of error in case the parameter isn't included. Given that we aren't deadset on this feature, I figured having that sort of flexibility wouldn't be too detrimental.

And understood. I certainly figured that the tsv's would be rather excessive if included.
To verify, is there some way I can edit my pull request to ensure those aren't included, or should I just be mindful in future commits to not include those files?

Reply:
I don't have a local checkout of this branch, so I think you should be safe to delete your last commit via resetting the head and force-pushing the branch. Or you just revert the commit (which keeps it in the history, with another undo-commit). In both cases, you can then make another commit on the branch which changes only the needed files.

Reply:
I believe the latest commit was properly reverted and should have only the necessary changes, but let me know if I didn't do it correctly.